pylutron_caseta/smartbridge.py

```diff
@@ -140,10 +143,25 @@ def add_occupancy_subscriber(
         """
         self._occupancy_subscribers[occupancy_group_id] = callback_
 
+    def add_button_subscriber(
+        self, button_id: str, callback_: Callable[[], None]
```

Comment:
```suggestion
        self, button_id: str, callback_: Callable[[str], None]
```
This event handler receives whether the button was pressed or released as a parameter.

Reply:
I'll make this change